src/router/router.tsx

```diff
@@ -33,21 +33,35 @@ export default function Router() {
     },
     {
       path: DEFAULTLAYOUT_PATH,
-      element: <DefaultLayout />,
-      children: [
-        {
-          path: VENDORS_PATH,
-          element: <Vendors />
-        },
-        {
-          path: ZONES_PATH,
-          element: <Zones />
-        },
-        {
-          path: MARKERS_PATH,
-          element: <Markers />
-        }
-      ]
```

Comment:
lets revert back to this one where we had `element` for the layout and inside the paths with components

Reply:
try now mate